src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -1,378 +1,420 @@
+              (String) data[10])
+              .getUrlForPath((String) data[1]),
+          (String) data[9]);
     }
+  }
 
-    @Test(groups = {"TestUrlBuilder", "unitTests"})
-    public void getWikiaUrlTests() throws Exception{
-        Constructor<UrlBuilder> c = getUrlBuilderConstructor();
+  @Test(groups = {"TestUrlBuilder", "unitTests"})
+  public void getWikiaUrlTests() throws Exception {
+    Constructor<UrlBuilder> c = getUrlBuilderConstructor();
 
-        Assert.assertEquals(c.newInstance("mediawiki119","preview", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
-                "http://www.preview.wikia.com");
-        Assert.assertEquals(c.newInstance("mediawiki119","sandbox-s1", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
-                "http://www.sandbox-s1.wikia.com");
-        Assert.assertEquals(c.newInstance("mediawiki119","prod", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
-                "http://www.wikia.com");
-    }
+    Assert.assertEquals(c.newInstance("mediawiki119", "preview", false, false, DEFAULT_LANGUAGE)
+            .getWikiGlobalURL(),
+        "http://www.preview.wikia.com");
+    Assert
+        .assertEquals(c.newInstance("mediawiki119", "sandbox-s1", false, false, DEFAULT_LANGUAGE)
+                .getWikiGlobalURL(),
+            "http://www.sandbox-s1.wikia.com");
+    Assert.assertEquals(c.newInstance("mediawiki119", "prod", false, false, DEFAULT_LANGUAGE)
+            .getWikiGlobalURL(),
+        "http://www.wikia.com");
+  }
 
-    private Constructor<UrlBuilder> getUrlBuilderConstructor() throws NoSuchMethodException {
-        Constructor<UrlBuilder> c = UrlBuilder.class
-                .getDeclaredConstructor(String.class, String.class, Boolean.class, Boolean.class, String.class);
-        c.setAccessible(true);
-        return c;
-    }
+  private Constructor<UrlBuilder> getUrlBuilderConstructor() throws NoSuchMethodException {
+    Constructor<UrlBuilder> c = UrlBuilder.class
+        .getDeclaredConstructor(String.class, String.class, Boolean.class, Boolean.class,
+            String.class);
+    c.setAccessible(true);
+    return c;
+  }
 
-    @Test(groups = {"TestUrlBuilder", "unitTests"})
-    public void getUrlForPathTests() throws Exception {
-        Constructor<UrlBuilder> c = getUrlBuilderConstructor();
-        
-        Assert.assertEquals(c.newInstance("mediawiki119","preview", false, false, DEFAULT_LANGUAGE).getUrlForPath("/wiki/Special:Version"),
-               "http://mediawiki119.preview.wikia.com/wiki/Special:Version");
+  @Test(groups = {"TestUrlBuilder", "unitTests"})
+  public void getUrlForPathTests() throws Exception {
+    Constructor<UrlBuilder> c = getUrlBuilderConstructor();
 
-        String cb = "cb=1111";
-        Configuration.setTestValue("qs", cb);
+    Assert.assertEquals(c.newInstance("mediawiki119", "preview", false, false, DEFAULT_LANGUAGE)
+            .getUrlForPath("/wiki/Special:Version"),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/wiki/Special:Version" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)